internal/pkg/remote/endpoint/service.go

```diff
@@ -28,18 +28,18 @@ const defaultTimeout = 10 * time.Second
 // Default Sylabs cloud service endpoints.
 const (
 	SCSDefaultCloudURI     = "cloud.sylabs.io"
-	DefaultApptainerHost   = "ghcr.io"
-	DefaultRegistryURI     = "https://ghcr.io"
+	SCSDefaultLibraryURI   = "https://library.sylabs.io"
```

Comment:
@DrDaveD @gmkurtzer not sure all of our unit tests should depend on 3rd party commercial server -- this also regresses our newer GHCR server for containers